packages/sync-actions/test/customer-group-sync.spec.js

```diff
@@ -1,38 +1,136 @@
 import customerGroupSyncFn, { actionGroups } from '../src/customer-group'
 import { baseActionsList } from '../src/customer-group-actions'
 
-describe('Exports', () => {
+describe('Discount Groups Exports', () => {
   it('action group list', () => {
-    expect(actionGroups).toEqual(['base'])
+    expect(actionGroups).toEqual(['base', 'custom'])
   })
 
-  it('correctly defined based actions list', () => {
-    expect(baseActionsList).toEqual([
-      { action: 'changeName', key: 'name' },
-      { action: 'setKey', key: 'key' },
-    ])
+  describe('action list', () => {
+    it('should contain `changeName` action', () => {
+      expect(baseActionsList).toEqual(
+        expect.arrayContaining([{ action: 'changeName', key: 'name' }])
+      )
+    })
+
+    it('should contain `setKey` action', () => {
+      expect(baseActionsList).toEqual(
+        expect.arrayContaining([
+          {
+            action: 'setKey',
+            key: 'key',
+          },
+        ])
+      )
+    })
   })
 })
 
-describe('Actions', () => {
+describe('Discount Groups Actions', () => {
   let customerGroupSync
   beforeEach(() => {
     customerGroupSync = customerGroupSyncFn()
   })
 
-  it('should build `changeName` action', () => {
-    const before = { name: 'the-greatest-name-before' }
-    const now = { name: 'the-greatest-name-now' }
-    const expected = [{ action: 'changeName', ...now }]
+  it('should build the `changeName` action', () => {
+    const before = {
+      name: { en: 'en-name-before', de: 'de-name-before' },
+    }
+
+    const now = {
+      name: { en: 'en-name-now', de: 'de-name-now' },
+    }
+
+    const expected = [
+      {
+        action: 'changeName',
+        name: { en: 'en-name-now', de: 'de-name-now' },
+      },
+    ]
     const actual = customerGroupSync.buildActions(now, before)
     expect(actual).toEqual(expected)
   })
 
-  it('should build `setKey` action', () => {
-    const before = { key: 'unique-key-before' }
-    const now = { key: 'unique-key-now' }
-    const expected = [{ action: 'setKey', ...now }]
+  it('should build the `setKey` action', () => {
+    const before = {
+      key: 'foo-key',
+    }
+
+    const now = {
+      key: 'bar-key',
+    }
+
+    const expected = [
+      {
+        action: 'setKey',
+        key: 'bar-key',
+      },
+    ]
+    const actual = customerGroupSync.buildActions(now, before)
+    expect(actual).toEqual(expected)
+  })
+
+  describe('custom fields', () => {
+    it('should build `setCustomType` action', () => {
+      const before = {
+        custom: {
+          type: {
+            typeId: 'type',
```

Comment:
I just realised that this is slightly different from `reference`. Let's use a real example here?
the [list is rich](http://dev.commercetools.com/http-api-projects-custom-fields.html#customizable-resources) to pick from :)

Reply:
This is a copy from elsewhere. Not sure if I'd rather keep it "consistent" in that sense.

Reply:
@adnasa The `type` in the `custom` object is in fact, [a reference to a type](http://dev.commercetools.com/http-api-projects-custom-fields.html#customfields)

Reply:
I did write (but discarded) that it is easy to confuse custom-type to reference :)
thanks for the info 👍